packages/venia-concept/src/components/MiniCart/miniCart.js

```diff
@@ -65,24 +76,56 @@ class MiniCart extends Component {
                     </Trigger>
                 </div>
                 <div className={classes.body}>
-                    <ProductList items={data} />
+                    {cartId ? (
+                        <ProductList
+                            currencyCode={cartCurrencyCode}
+                            items={cart.details.items}
+                        />
+                    ) : null}
```

Comment:
Please don't do multiline expressions in JSX, it defeats the purpose of the syntax. Consider [using a getter](https://github.com/magento-research/pwa-studio/blob/master/packages/venia-concept/src/components/ProductOptions/option.js#L25-L27) or a helper method, then [inject the result directly](https://github.com/magento-research/pwa-studio/blob/master/packages/venia-concept/src/components/MiniCart/miniCart.js#L82), as seen in this file and others.